dist/comment_style.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import argparse, os, re, subprocess, sys
+
+def print_msg(file_name, line_num, line, multiline):
+    print("Illegal " + multiline + "comment in " + file_name + ":" + str(line_num - 1) + " " +
+        line.strip('\n'))
+
+def check_c_comments(file_name):
+    count = 0
+    with open(file_name) as f:
+        line_num = 0
+        for line in f:
+            stripped = line.strip()
+            if match(stripped):
+                print_msg(file_name, line_num, line, "")
+                count +=1
+            line_num += 1
+    return count
+
+def match(stripped_line):
+    # We don't worry about whitespace in matches as we strip the line of whitespace.
+    url = re.compile(r'https?:\/\/')
```

Comment:
I'd add a comment here explaining that `//` as part of a URL is a false positive we want to ignore.

Reply:
Thanks, done in bf9d525.